testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -5727,10 +5727,32 @@ private static void checkIf(SqlOperatorFixture f) {
     f.checkQuery("select regexp_instr('a9cadca5c4aecghi', 'a[0-9]c', 1, 3)");
   }
 
-  @Test void testRegexpReplaceFunc() {
+  @Test void testRegexpReplace2Func() {
     final SqlOperatorFixture f0 = fixture();
     final Consumer<SqlOperatorFixture> consumer = f -> {
-      f.setFor(SqlLibraryOperators.REGEXP_REPLACE);
+      f.setFor(SqlLibraryOperators.REGEXP_REPLACE_2);
```

Comment:
I am assuming these were validated against postgres. If true, please add a comment.

Reply:
I took another look and realized that it is only Amazon Redshift that supports a 2 argument signature. I have updated accordingly. I have also tested other engines.
* With 4 arguments, all engines expect a signature of "string, string, string, int"
* Oracle only supports the signature "string, string, string, int, int" for 5 arguments
* MySQL and Amazon Redshift support two signature for 5 arguments
  * "string, string, string, int, int"
  * "string, string, string, int, string"

Updated to better match the signatures each engine supports.